7.sql

```diff
@@ -0,0 +1,46 @@
+USE banking_db
+GO
+
+CREATE OR ALTER PROCEDURE MoneyTransfer (@inputMoneyToMove MONEY = 3, @inputUserName NVARCHAR(30) = 'GGD', @inputCardName NVARCHAR(30) = 'VISA Classic')

```

Comment:
the inputUserName parameter is excess, try to implement the procedure without it